main.go

```diff
@@ -18,40 +25,74 @@ import (
 )
 
 func main() {
+	log.SetFlags(log.Flags() | log.Lshortfile)
+	log.SetOutput(os.Stderr)
```

Comment:
Sure! I've only looked into this briefly, but here's where I ended up. Either way, I don't want to disrupt what is currently in place consuming the logs, so happy to change it to stdout.

- stdout: is for application output.
- stderr: is for application logging.

I'm defining "application _output_" as the result of some operation or the question I'm asking. "application _logging_" then, is essentially _everything else_.

For example, if I were to write a calculator program to add two numbers, I would return the result of that operation, the application output, to stdout. Other application logging, like startup, error handling, etc, would go to stderr. The idea being that if I were to run the calculator and pipe it's output into another program, the application logging wouldn't get picked up by the pipe and fed into the next program.

In the case of a web server, like this, we're probably not generating "output" and would be generating only logging, and since that's the case, we can choose either one to consolidate all of our logs at. Whatever works best for the application scooping up and aggregating the logs.

Another hint is in the default behaviors of the Go stdlib.
- `log` [The default output is stderr](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/log/log.go;l=87)
- `fmt` [The default output is stdout](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/fmt/print.go;l=241-243)

So technically speaking, I don't even need to specify that I want it to go to `stderr`, it would do that by default. I am doing it here though, in order to be explicit, and also to override any `init` functions that may have gotten a bit grabby, [which is actually another problem altogether](https://gist.github.com/kevinfalting/6b6c6e989be71e114ecd89437c5e10f4).

Reply:
Thanks for that info! In most cloud native applications, both `stdout` and `stderr` are consolidated, so it shouldn't make a difference here. I confirmed in the [fly docs](https://fly.io/docs/reference/postgres/#logs) that this is the case there as well (the page references their Postgres service but I'm 99% sure it applies across the board).

I wasn't aware of those defaults in the Go stdlib though, definitely informative.